examples/labelelide-example.ts

```diff
@@ -0,0 +1,308 @@
+as I lie close to the earth, a thousand unknown plants are noticed by me: when I hear the buzz of the little world \
+among the stalks, and grow familiar with the countless indescribable forms of the insects and flies, then I feel the \
+presence of the Almighty, who formed us in his own image, and the breath of that universal love which bears and \
+sustains us, as it floats around us in an eternity of bliss;  and then, my friend, when darkness overspreads my eyes, \
+and heaven and earth seem to dwell in my soul and absorb its power, like the form of a beloved mistress, then I often \
+think with longing, Oh, would I could describe these conceptions, could impress upon paper all that is living so full \
+and warm within me, that it might be the mirror of my soul, as my soul is the mirror of the infinite God!';
+
+        this._labelSize = new Position2DLabel(new Text(), Label.Type.Dynamic);
+        this._labelSize.fontSize = 20;
+        this._labelSize.fontSizeUnit = Label.Unit.Pixel;
+        this._labelSize.alignment = Label.Alignment.Left;
+        this._labelSize.lineAnchor = Label.LineAnchor.Baseline;
+        this._labelSize.color.fromHex('#27aae1');
+
+        this._labelLeft = new Position2DLabel(
+            new Text(`Label.Elide.Right |  ${werther}`), Label.Type.Dynamic);
+        this._labelLeft.fontSizeUnit = Label.Unit.Pixel;
+        this._labelLeft.elide = Label.Elide.Right;
+        this._labelLeft.alignment = Label.Alignment.Left;
+        this._labelLeft.lineAnchor = Label.LineAnchor.Baseline;
+        this._labelLeft.color.fromHex('#fff');
+
+        this._labelRight = new Position2DLabel(
+            new Text(`${werther}  | Label.Elide.Left`), Label.Type.Dynamic);
+        this._labelRight.fontSizeUnit = Label.Unit.Pixel;
+        this._labelRight.elide = Label.Elide.Left;
+        this._labelRight.alignment = Label.Alignment.Right;
+        this._labelRight.lineAnchor = Label.LineAnchor.Baseline;
+        this._labelRight.color.fromHex('#fff');
+
+        this._labelMiddle = new Position2DLabel(
+            new Text(`Label.Elide.Middle |  ${werther}  | Label.Elide.Middle`), Label.Type.Dynamic);
+        this._labelMiddle.fontSizeUnit = Label.Unit.Pixel;
+        this._labelMiddle.elide = Label.Elide.Middle;
+        this._labelMiddle.alignment = Label.Alignment.Center;
+        this._labelMiddle.lineAnchor = Label.LineAnchor.Baseline;
+        this._labelMiddle.color.fromHex('#fff');
+
+        this._labelCustom = new Position2DLabel(
+            new Text(`Custom Ellipsis |  ${werther}`), Label.Type.Dynamic);
+        this._labelCustom.fontSizeUnit = Label.Unit.Pixel;
+        this._labelCustom.elide = Label.Elide.Right;
+        this._labelCustom.alignment = Label.Alignment.Left;
+        this._labelCustom.lineAnchor = Label.LineAnchor.Baseline;
+        this._labelCustom.color.fromHex('#fff');
+
+
+        this._labelPass.aaStepScale = 1.0;
+        // this._labelPass.sampling = LabelRenderPass.Sampling.
```

Comment:
is this commented out because it does not work? or it is not part of the example? This line has bad syntax, so maybe fix that?